frontend/src/metabase/common/components/FilterPicker/StringFilterPicker/StringFilterPicker.tsx

```diff
@@ -77,17 +78,29 @@ export function StringFilterPicker({
     );
   };
 
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
```

Comment:
I can't submit the form with Enter when excluding extraction units


https://github.com/metabase/metabase/assets/6830683/ee918677-6e8e-4b83-ac8a-c47a6d5447d2